dist/husky.js

```diff
@@ -51124,6 +51152,19 @@ define('husky_extensions/util',[],function() {
                 return $.extend(true, parent, object);
             };
 
+            /**
+             * Returns a parameter value from a given url
```

Comment:
http://stackoverflow.com/questions/901115/how-can-i-get-query-string-values-in-javascript